app/javascript/chat/RequestManager/RequestManager.jsx

```diff
@@ -130,7 +130,7 @@ export default class RequestManager extends Component {
     const { channelJoiningRequests, userInvitations } = this.state;
 
     return (
-      <div className="activechatchannel__activeArticle activesendrequest">
+      <div className="activesendrequest">
```

Comment:
I looked through the code base and the `activesendrequest` CSS class is not defined anywhere in the code base. The only reference to is in this file and channelRequest.jsx.

@narender2031 @sarthology, is this class constructed somewhere in JS or is this class no longer required?

![image](https://user-images.githubusercontent.com/833231/95911880-09fce900-0d70-11eb-984f-8d74f07911a4.png)